### ISSUE TITLE
您好，我在使用test.py时发现有一些图片的输出是长度为2的一维张量，这是为什么呢

### ISSUE THREAD

**Issue by chenxia00** [yangyuke001/FIIQA-PyTorch#10]
facePath = imagePath + "/" + img
    face = Image.open(facePath)
    imgblob = dataTransforms(face).unsqueeze(0)
    imgblob = Variable(imgblob)
    torch.no_grad()
    predict = F.softmax(net(imgblob), dim=1)
predict是一个长度为2的一维张量，正常情况下是长度为1的一维张量。
这是我的邮箱，1658550916@qq.com。需要的话，我可以将这样的图片发给您。
